e2e-tests/capabilities-add-drop-build.yaml

```diff
@@ -20,12 +20,18 @@ environment:
 pipeline:
   - name: Test default effective capability
     runs: |
+      # Skip test if on QEMU runner, since the runner does not support process capabilities Melange feature.
+      { test -f /sys/class/dmi/id/sys_vendor && grep -E "^QEMU$" /sys/class/dmi/id/sys_vendor && exit 0; } || true
```

Comment:
Hmm, what isn't supported?  cc @89luca89

Reply:
@mattmoor because the capabilities add and drop feature has been implemented for the Docker and Bubblewrap runners only
* https://github.com/chainguard-dev/melange/blob/90c96db19b0b2a05d98fa7bc90b8775edeb0ddd1/pkg/container/bubblewrap_runner.go#L136
* https://github.com/chainguard-dev/melange/blob/90c96db19b0b2a05d98fa7bc90b8775edeb0ddd1/pkg/container/docker/docker_runner.go#L102

Reply:
cc @antitree as well

Reply:
@antitree @mattmoor we can open a separate issue to track if we want cap-add/drop in QEMU I think and tackle it later